### DISCUSSION TITLE
Is it possible to set up the Roccat Kone Aimo buttons?

### DISCUSSION THREAD

**Discussion by Marethyu61** [X3n0m0rph59/eruption#98] [Q&A] [answered]
Hello,

I would like to thank you for creating this module that allows me to change the colors on my Kone Aimo. There was roccat-tools, but the module was old and did not work on my mouse. Although I have a low level on Linux, I managed to install it with your instructions which were clear and precise.

I wanted to know if it is possible to configure the buttons of my mouse with the software. And if it is not possible, is it planned in the long run (no need to rush, it can take several months if necessary) to be able to manage them from Eruption?

Thanks in advance

Marethyu

PS : English is not my first language, sorry if there are mistakes.

**Comment by X3n0m0rph59** [comment, marked as answer]
Hi @Marethyu61, thanks for posting this question!

Configuring the hardware profiles of the `ROCCAT Kone Aimo` is currently not implemented in Eruption, this means that you can not currently get the mouse to report different keyboard or mouse events on button clicks outside of Eruption/when Eruption is not running. 

I believe it could be possible to assign hardware profiles on Windows using ROCCAT Swarm and carry it over to Linux... Or maybe libratbag/Piper does support your hardware: [libratbag GitHub](https://github.com/libratbag/libratbag/)

When the hardware profile feature is not strictly required, you could implement any kind of functionality using Lua macros today, that get executed as long as Eruption is running. This may be even more flexible than using hardware based profiles, since anything that can be done from Lua can be triggered by a keyboard or mouse event.

Support for device specific configuration and hardware profiles in Eruption will be added some time in the future.

**Comment by Marethyu61** [comment]
Hi,

All right, I will try the libratbag module. Thanks for the answer!

EDIT : Unfortunately, Roccat kone aimo is not taken into account by Libratbag. This is not a problem. Thanks anyway!

Marethyu

**Comment by madprops** [comment]
So I'm guessing mouse rebinding won't work on the Kova AIMO either :/
